magenta_docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Uses one of AI Platform base images.
+# You can try using different images however only this one has been tested.
+FROM gcr.io/deeplearning-platform-release/tf2-gpu.2-2
+WORKDIR /root
+
+# Installs sndfile library for reading and writing audio files.
+RUN apt-get update && apt-get install -y libsndfile-dev
```

Comment:
Use `--no-install-recommends` too to avoid installing unintended packages.

Reply:
Done